pandas/core/arrays/string_arrow.py

```diff
@@ -752,3 +753,10 @@ def _str_map(self, f, na_value=None, dtype: Dtype | None = None):
             #    or .findall returns a list).
             # -> We don't know the result type. E.g. `.get` can return anything.
             return lib.map_infer_mask(arr, f, mask.view("uint8"))
+
+    def _str_contains(self, pat, case=True, flags=0, na=np.nan, regex=True):
+        if not regex and case:
+            result = pc.match_substring(self._data, pat)
```

Comment:
This should have additionally a pyarrow version or `hasattr(pc, ..)` check?

Reply:
pretty sure match_substring is available in pyarrow 1.0.0 (our min for the creation of ArrowStringArray) otherwise tests would fail.